Daytime-Operations/Auxiliary-Telescope/AT-Calibrations/prepare-atcs-for-flat-fields-procedure.rst

```diff
@@ -8,7 +8,79 @@
 Prepare ATCS For Flat Fields Procedure
 ######################################
 
-Under construction.
+Overview
+========
+
+This procedure describes how to execute flat field calibration with LATISS during day time checkout procedure, using different filters.
+This is the default end-of-checkout script.
+NOTE: Prepare to take fltas only if you have sufficient time to vent the telescope afterwards.
+
+It will leave the telescope in position to start taking afternoon calibraitons. This is az = 205.7 elev = 30.
+If you are not sure how you want to leave the telescope, consider running this script as it will perform a few more useful checks of the system.
+
+Prerequisites
+=============
+Be sure you have run previously the following scripts in the :ref:`daytime checkout <AT-Daytime-Checkout-Daytime-Checkout>`. in ATQueue-love:
+
+
+
+.. figure:: ../_static/love-mtqueue-atqueue-panel.png
+     :name:  latiss-available-scripts-love
+
+     LOVE interface with the "ATQueue" pannel.
+
+
+
+.. figure:: ../_static/love-available-scripts.png
+     :name:  latiss-available-scripts-love
+      
+     LOVE interface with the "AVAILABLE SCRIPTS" list.
+
+    
+
+- auxtel/enable_latiss.py 
+- auxtel/daytime_checkout/latiss_checkout.py
+- auxtel/enable_atcs.py 
+- autel/daytime_chekout/atpnuematics_checkout.py 
+- auxtel/daytime_checlout/telescope_and_dome_checkout.py
+- auxtel/daytime_checkout/slew_and_take_image_checkout.py
+
+Home the dome to make sure it knows where it is.
+
+Post-Condition
+==============
+Telescope is ready to :ref:`take calibrations <LATISS-Combined-Calibrations-Procedure-LATISS-Combined-Calibrations-Generation-Procedure>`.
+
+NOTE: make sure M3 is pointing to LATISS, mirror cover are is opened and the telescope is in position, as a final visual check.
+
+Procedure Steps
+===============
+Run: auxtel/daytime_checkout/prepare_for/flat.py script:
```

Comment:
Code snippets are rendered in a way that is more obvious like this:

Run the `auxtel/daytime_checkout/prepare_for/flat.py` script.